app/helpers/error_helper.rb

```diff
@@ -0,0 +1,37 @@
+module ErrorHelper
+  # _path & _url helpers are usually available in instance, not class scope.
+  URL_HELPERS = route_url_helpers = Rails.application.routes.url_helpers
```

Comment:
Why are we assigning to `route_url_helpers` here?

Reply:
Because I made a mistake that my main man @ktp242 later corrected in master in https://github.com/NYPL/scsbuster/commit/d1f061c375f96a5de1cae3ba046c2df175fe3980